source/getting-started/installation-raspberry-pi-all-in-one.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: "Raspberry Pi All-In-One Installer"
+date: 2016-05-12 01:39
+comments: true
+sharing: true
+footer: true
+---
+Easily deploy a complete Home-Assistant server, complete with Websocket MQTT and Z-Wave support out of the box using Fabric!
+
+**Usage:**
+*Start with a fresh Raspbian Jessie/Jessie-Lite or Debian 8 install*
+ 1. install fabric locally: ```pip3 install fabric3```
+ 2. Ensure you can SSH into Target Host
+ 3. Clone the repo : ``` git clone https://github.com/jbags81/fabric-home-assistant.git ``` on  your local host. (The repo contains a pre-configured default mosquitto.conf file. The only addition is an added listener for websockets listening on 9001. It also contains preconfigured systemd service profiles. For the fabric script to run successfully, it has to be ran from the root of the cloned repo.)
+ 4. Edit ```fabfile.py``` and add the host info for the target host.
+ 5. Run the "deploy" job to build home-assistant server: ``` fab deploy ```
+ 6. Reboot
+
+
+ Everything will start at boot, and Home-Assistant is accessible now from **http://your_server_ip:8123**
+
+ The Home-Assistant configs are located in: ```/home/hass``` The virtualenv path, along with where all python packages will install is located at: ```/srv/hass/hass_venv```
+
+ <hr>
+**The All-In-One Fabric script will do the following automatically:**
```

Comment:
Because @balloob told me to push the instructions to the top. You two are in conflict on your direction...